content/applications/productivity/iot/config/windows_iot.rst

```diff
@@ -0,0 +1,112 @@
+===============================================================
+Use Windows IoT software to connect devices to an Odoo database
+===============================================================
+
+A Virtual IoT box is a computer program that needs to be downloaded and installed on a Windows
+computer. This requires a Windows operating system with an Odoo 16 or later database.
+
+The Windows Virtual :abbr:`IoT (Internet of Things)` box works the same way as a physical
+:abbr:`IoT (Internet of Things)` box, with the ability to run most of the same devices. All
+:abbr:`POS (Point of Sale)` devices work with it, such as a scale or printer. Payment terminals will
+also work, but it should be noted that :abbr:`MRP (Material Requirement Planning)` devices are not
+compatible. These include cameras or measurement tools.
+
+Pre-requisites
+==============
+
+- Odoo 16 database
+- :abbr:`IoT (Internet of Things)` compatible devices
```

Comment:
Might be worth to add a .. seealso to the section with iot devices (or to the compatibility table I mentioned earlier)

Reply:
I've added the following: 

`- Odoo 16 database or any version above
- :abbr:`IoT (Internet of Things)` compatible devices

  ..seealso::
    `Odoo's compatible IoT devices <https://www.odoo.com/app/iot-hardware>`_`